docs/quickstart/index.md

```diff
@@ -6,20 +6,162 @@ layout: docs
 type: markdown
 sidenav: doc-side-quickstart-nav.html
 ---
-<h2 class="top">Get started with Spine</h2>
+<h2 class="top">Getting Started with Spine</h2>
 
 <div id="toc" class="toc hide-block"></div>
 
-These pages show you how to get up and running in Spine.
-There is a Quick Start for a minimal server-side and client-side applications written in Java.
+This guide helps you get started with Spine by creating a server-side and client-side applications (written in Java).
 
-There are also guides for clint-side applications written in other supported languages.
-In order to run these examples, you'll need the Java server-side application.
+<p class="note">We recommend familiarizing with [Spine basic concepts]({{site.baseurl}}/docs/guides/concepts.html) before getting started with sample applications.</p>
 
-For an overview of some of the core concepts, see [Spine Concepts]({{site.baseurl}}/docs/guides/concepts.html).
-For more tutorials and examples, see our [Tutorials]({{site.baseurl}}/docs/tutorials).
+For more tutorials and examples, see the [Tutorials]({{site.baseurl}}/docs/tutorials) section for other supported languages:
 
  - [Java](java.html)
  - [JavaScript](javascript.html)
  - [C++](cpp.html)
 
+<p class="note">Please note you will need a Java server-side application to run these examples.
```

Comment:
This application _contains_ a server-side application. But in reality a person would run both client- and server-side apps locally on his workstation. Nobody is going to deploy this toy example to a real server.

Why do we need this note at all? What do we want to say and why?